superset-frontend/src/components/Chart/DrillBy/DrillByMenuItems.tsx

```diff
@@ -218,7 +228,7 @@ export const DrillByMenuItems = ({
                   key={`drill-by-item-${column.column_name}`}
                   tooltipText={column.verbose_name || column.column_name}
                   {...rest}
-                  onClick={() => openModal(column)}
+                  onClick={() => openModal(null, column)}
```

Comment:
Can we pass the event from onClick to openModal instead of null? Like `onClick={e => openModal(e, column)}`.

That will require fixing the typing in `MenuItemWithTruncation` though - from `onClick?: () => void` to `onClick?: MenuProps['onClick'];` (`import type { MenuProps } from 'antd/lib/menu';`)